Tasks/DownloadBuildArtifactsV0/main.ts

```diff
@@ -136,30 +136,46 @@ async function main(): Promise<void> {
             }
         }
 
+        // if the definition name includes a variable then definitionIdSpecified is a name vs a number
+        if (Number.isNaN(parseInt(definitionIdSpecified))) {
+            var definitions: BuildDefinitionReference[] = await executeWithRetries("getBuildDefinitions", () => buildApi.getDefinitions(projectId, definitionIdSpecified), retryLimit).catch((reason) => {
+                reject(reason);
+                return;
+            });
+
+            if (!definitions || definitions.length < 1) {
+                reject(tl.loc("InvalidBuildDefinitionID", definitionId));
+                return;
+            }
+
+            definitionId = String(definitions[0].id);
+            console.log(tl.loc("DefinitionIDFound", definitionIdSpecified, definitionId));
```

Comment:
rename macro to DefinitionNameMatchFound?

Reply:
changed